AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC.xcodeproj/project.pbxproj

```diff
@@ -534,6 +575,65 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
```

Comment:
Actually it looks like you also committed some changes that are added automatically by `pod install` which we didn't have before. Curious @thomasmso if you think this is fine to leave in or if we should remove it

Reply:
It's fine, this always comes up and ends up being more work for us to manually remove from the commit